zcash_primitives/src/transaction/components/transparent/pczt.rs

```diff
@@ -0,0 +1,217 @@
+//! PCZT support for transparent Zcash.
+
+use std::collections::BTreeMap;
+
+use bip32::ChildNumber;
+use getset::Getters;
+use zcash_protocol::value::Zatoshis;
+
+use crate::{
+    legacy::Script,
+    transaction::{sighash::SighashType, TxId},
+};
+
+mod parse;
+pub use parse::ParseError;
+
+mod updater;
+pub use updater::{InputUpdater, OutputUpdater, Updater, UpdaterError};
+
+mod signer;
+pub use signer::SignerError;
```

Comment:
Should this be behind a feature flag to avoid an unconditional dependency on secp256k1? It doesn't currently build without "transparent-inputs".
```suggestion
#[cfg(feature = "transparent-inputs")]
mod signer;
#[cfg(feature = "transparent-inputs")]
pub use signer::SignerError;
```